<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сравнение моделей данных Вертикаль</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="logPlainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>519</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openFirstFileAction"/>
    <addaction name="openSecondFileAction"/>
    <addaction name="saveLogFileAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Действия</string>
    </property>
    <addaction name="diffAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openFirstFileAction"/>
   <addaction name="openSecondFileAction"/>
   <addaction name="diffAction"/>
   <addaction name="saveLogFileAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openFirstFileAction">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icons/open1.svg</normaloff>:/icons/open1.svg</iconset>
   </property>
   <property name="text">
    <string>Открыть первый файл...</string>
   </property>
  </action>
  <action name="openSecondFileAction">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icons/open2.svg</normaloff>:/icons/open2.svg</iconset>
   </property>
   <property name="text">
    <string>Открыть второй файл...</string>
   </property>
  </action>
  <action name="saveLogFileAction">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icons/save.svg</normaloff>:/icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Сохранить различия...</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icons/exit.svg</normaloff>:/icons/exit.svg</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="diffAction">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/icons/run.svg</normaloff>:/icons/run.svg</iconset>
   </property>
   <property name="text">
    <string>Сравнить</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
